angular/src/lib/modal/examples/modal-examples.module.ts

```diff
@@ -1,22 +1,30 @@
-import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import {
-  ModalBodyModule,
-  ModalFooterModule,
-  ModalHeaderModule,
-  ModalModule
-} from '@momentum-ui/angular';
-import { ExampleModalDefaultComponent } from './index';
+import { NgModule } from '@angular/core';
+import { ButtonModule } from '@momentum-ui/angular';
+import { ModalModule } from '../modal.module';
+
+import { ExampleModalDefaultComponent } from './modal-default.component';
+import { ExampleModalSmallComponent } from './modal-small.component';
+import { ExampleModalLargeComponent } from './modal-large.component';
+import { ExampleModalFullComponent } from './modal-full.component';
 
```

Comment:
Can we import all these ExampleComponents from the './index' instead of individual imports. Looks like we've been doing it a mix of either across components. Perhaps we should establish a common pattern for importing examples if we haven't already. Thoughts? @momentum-design/collab-jedi 

import {
  ExampleModalDefaultComponent,
  ExampleModalSmallComponent,
  ExampleModalLargeComponent,
   } from './index';

Reply:
we do it this way to make them available in the debugger, otherwise they are not there